src/Utils.php

```diff
@@ -674,20 +674,6 @@ function wf_civicrm_api($entity, $operation, $params) {
     return $result;
   }
 
-  /**
-   * Adjust Metadata for Transact action.
-   *
-   * The metadata is used for setting defaults, documentation & validation.
-   *
-   * @param array $params
-   *   Array of parameters determined by getfields.
-   */
-  function _wf_civicrm_api3_contribution_transact_spec(&$params) {
-    $fields = civicrm_api3('Contribution', 'getfields', ['action' => 'create']);
-    $params = array_merge($params, $fields['values']);
-    $params['receive_date']['api.default'] = 'now';
-  }
-
```

Comment:
@KarinG Why did you include this in the original PR? I don't think it is relevant?

Reply:
Oh that was just a copy/paste. I noticed you had that in your replacement Extension as well and figured it may be useful.

Reply:
It won't do anything except in the context of API3 so not needed here.